racket/src/racket/sconfig.h

```diff
@@ -378,6 +378,33 @@
 
 # define FLAGS_ALREADY_SET
 
+#endif
+
+  /************** HAIKU with clang/gcc ****************/
+
+#if defined(__HAIKU__)
+
+# if defined(__i386__) || defined(__BEPC__)
+#  define SCHEME_PLATFORM_LIBRARY_SUBPATH "i386-haiku"
+#  define REGISTER_POOR_MACHINE
+#  define MZ_USE_JIT_I386
+#  define MZ_TRY_EXTFLONUMS
+# elif defined(__amd64__)
+#  define SCHEME_PLATFORM_LIBRARY_SUBPATH "amd64-haiku"
+#  define REGISTER_POOR_MACHINE
+#  define MZ_USE_JIT_X86_64
+#  define MZ_TRY_EXTFLONUMS
+# else
+#  error Unported platform.
```

Comment:
`Unported` sounds quite unusual to me. Did you mean `Unsupported` instead?

Reply:
Take this comment with a pinch of salt - I am not a native speaker so it might be perfectly fine.

Reply:
Probably a copy-paste typo... Will fix.